_test/test_sqw/test_gen_sqw_accumulate_sqw_parpool.m

```diff
@@ -51,38 +51,38 @@
             if ~exist('test_name','var')
                 test_name = mfilename('class');
             end
-            obj = obj@gen_sqw_common_config(-1,1,'mpi_code','parpool');
+            obj = obj@gen_sqw_common_config(-1,1,'mex_code','parpool');
             obj = obj@gen_sqw_accumulate_sqw_tests_common(test_name,'parpool');
         end
         
-        %------------------------------------------------------------------
-        %         % Block of code to disable some tests for debugging Jenkins jobs
-        function test_gen_sqw(obj,varargin)
-            if is_jenkins && ispc
-                warning('test_gen_sqw disabled')
-                return
-            else
-                test_gen_sqw@gen_sqw_accumulate_sqw_tests_common(obj,varargin{:});
-            end
-        end
-        function test_accumulate_sqw14(obj,varargin)
-            if is_jenkins && ispc
-                warning('test_accumulate_sqw14 disabled')
-                return
-            else
-                test_accumulate_sqw14@gen_sqw_accumulate_sqw_tests_common(obj,varargin{:});
-            end
-            
-        end
-        function test_accumulate_and_combine1to4(obj,varargin)
-            if is_jenkins && ispc
-                warning('test_accumulate_and_combine1to4 disabled')
-                return
-            else
-                test_accumulate_and_combine1to4@gen_sqw_accumulate_sqw_tests_common(obj,varargin{:});
-            end
-            
-        end
+        %         %------------------------------------------------------------------
+        %         %         % Block of code to disable some tests for debugging Jenkins jobs
+        %         function test_gen_sqw(obj,varargin)
```

Comment:
the tests can be disables by renaming to e.g. `function DISABLED_test_gen_sqw(obj, varargin)` rather than commenting out large blocks of code.

Reply:
They are enabled back, so this is outdated